pandas/core/groupby.py

```diff
@@ -393,6 +406,38 @@ def indices(self):
         self._assure_grouper()
         return self.grouper.indices
 
+    @property
+    def inputconstructor(self):
+        return self._inputconstructor
+
+    @inputconstructor.setter
+    def inputconstructor(self, constructor):
+        self._inputconstructor = constructor
+
```

Comment:
this naming schema is not good.

something more along the lines of ``_1d_constructor``, ``_2d_constructor`` is much better